### ISSUE TITLE
Some formatting foo for the training site

### ISSUE THREAD

**Issue by ACharbonneau** [nih-cfde/training-and-engagement#45]
[This requires a bit of css knowledge](https://github.com/nih-cfde/training-and-engagement/blob/master/docs/source/_static/my-styles.css), or [possibly html knowledge](https://github.com/nih-cfde/training-and-engagement/blob/master/docs/source/_templates/layout.html):

updated color palette for admonitions:
- prereqs different from learningobjectives different from exercise different from tips differetnt from error/warning/etc, but don't clash with overall color palette
- improve forward/back buttons to move with page or live at bottom

**Comment by ACharbonneau** [comment]
Colors fixed. at least for now. The CFDE color scheme might be changing from the blue/purple thing, so might need to be updated again.

**Comment by ACharbonneau** [comment]
For the forward and back buttons, and honestly a lot of the site, this was "Amanda's first foray into trying to build a site from scratch." And it is janky. I'm starting to get an idea of which things I need to fix to make it work well, but I also don't have as much time to spend on it as I did a few weeks ago. One quick fix would be finding a template that has all the features we need, but that isn't a hand crafted monstrosity. Then it's just a matter of fixing the colors, which you know how to do now :)

If you wanted to try to fix the buttons specifically without looking for a good theme, my specific problem with them is that they stay at the top of the page, even when there's a lot of text. I figured out how to make the middle panel scroll, so that the two side menus stayed in place, which fixed it. But that made the mobile version render terribly. I couldn't figure out how to scroll and not break on smaller screens.
